specs/agents/otel-distribution.md

```diff
@@ -0,0 +1,65 @@
+
+## Terminology
+
+**Vanilla OpenTelemetry Distribution**: this is the "upstream" OpenTelemetry distribution that is maintained by the OpenTelemetry community.
+Implementation differs per platform, but it usually consists of an API/SDK and can also provide automatic instrumentation.
+
+**Elastic OpenTelemetry Distribution**: this is an OpenTelemetry distribution provided by Elastic that is derived from the 
+_Vanilla OpenTelemetry Distribution_.
+
+## General guidelines
+
+Those statements are guiding principles of the Elastic OpenTelemetry distributions, they should be considered more as advice than strict rules.
+
+Elastic OpenTelemetry distribution SHOULD ideally:
+- behave as drop-in replacements of their upstream counterparts
+- provide a simple setup and favor onboarding experience (aka "things should work by default").
+- avoid capturing potentially confusing data (see [system metrics](#system-metrics) example below).
+
+## Configuration
+
+Elastic OpenTelemetry distributions MAY override the default configuration.
+When doing so, user-configuration should remain consistent with vanilla distribution:
+- explicit user configuration SHOULD remain effective
+- overriden default configuration MUST have the ability to be restored to upstream default
+
+Elastic specific configuration items MUST be prefixed with `ELASTIC_OTEL_`.
+For example, the [inferred spans](#inferred-spans) feature is configured with `ELASTIC_OTEL_INFERRED_SPANS_*`.
+
+Elastic and platform specific configuration items must be prefixed with `ELASTIC_OTEL_${platform}|` to be consistent with
+the upstream `OTEL_${platform}_` prefix.
+
+For simplicity the configuration in this specification will use the "environment variable" syntax, some platforms like Java
+might also support other ways to configure.
+
+## Features
+
+### Inferred spans
+
+Supported platforms: [Java](https://github.com/elastic/elastic-otel-java/tree/main/inferred-spans)
+
+Configuration namespace: `ELASTIC_OTEL_INFERRED_SPANS_*`
+
+### System metrics
+
+Those metrics are usually captured using the collector running locally but in case where no collector is present, or a centralized
```

Comment:
```suggestion
These metrics are usually captured using the collector running locally but in case where no collector is present, or a centralized
```